AfaqShuaib09/Medium_backend
# Docker setup

# Docker Setup

Description: Added the following features

1. Used Docker Compose to setup Django Rest Application

Testing Instructions:

1. Clone Repository from [GitHub](https://github.com/AfaqShuaib09/Medium_backend) `git clone https://github.com/AfaqShuaib09/Medium_backend.git`
5. Change your current present working directory (pwd) to `cd Medium_backend`
6. Change your current branch from main to medium_backend : `git checkout docker_setup`
7. Change your current directory to project root directory `cd cd medium_backend`
8. For manually building up container `docker-compose bulid` (optional)
9. for the docker image start `docker-compose up` (it will automatically build the docker container)

> check only the last two commits of this PR because It is extended from the cors_setup branch which hasn't been reviewed yet
